### PR TITLE
adds pattern for minimizing deployed environments

### PR DESCRIPTION
**Pull request by camcash17**
A proposal for minimizing deployed environments as green patterns

### CODE REVIEW

**Comment by dubrie** [comment]
assigning to @dubrie for SME Review

**Comment by ravikp** [comment]
How about changing the title to "Repurpose the environments" instead of "Minimizing deployed environments"? Also include running automated performance tests in the night and doing functional testing in the environment in the morning? Just a thought.

File: docs/catalog/minimizing-deployed-environments.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Solution
+Minimise the amount of deployed environments, perhaps considering the necessity of a QA or Performance testing environment.
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+
+Concerning the SCI equation. Reducing the amount of deployed environments will impact one part:
+
+- `E`: We reduce the total electricity required by reducing provisioned resources.
+- `M`: By reducing the amount of nodes running on underlying machines, we reduce the total embodied carbon.
```
Comment (dubrie) [inline docs/catalog/minimizing-deployed-environments.md:30]: I think this should just focus on reducing the total number of machines and resources. Deployed environments don't typically reduce nodes but rather full machines/virtual machines.

**Review by dubrie — CHANGES_REQUESTED** [review]
One ask for update on this would be to align to the updated [Template.md file](https://github.com/Green-Software-Foundation/green-software-patterns/blob/dev/TEMPLATE.md). While this discussion was happening, we modified the template to include references and shift some of the items to a metadata section. It should be straight-forward but please let me know if there is any confusion on which data should go where in the new format.

**Comment by dubrie** [comment]
> How about changing the title to "Repurpose the environments" instead of "Minimizing deployed environments"? Also include running automated performance tests in the night and doing functional testing in the environment in the morning? Just a thought.

I think the title of this is accurate for the content, the goal of this pattern is to reduce and minimize the environments that you deploy and use. However, I think the other two options to propose with running automated test in the night and doing functional testing in the morning are interesting as a separate pattern. Is there a connection to the SCI Impact that you see? I've opened a new issue to continue the discussion further on this: https://github.com/Green-Software-Foundation/green-software-patterns/issues/85

**Comment by dubrie** [comment]
Pinging @camcash17 for updates and/or thoughts on the feedback.

**Pull request closed without merging**